lib/middleman-search_engine_sitemap/extension.rb

```diff
@@ -49,14 +41,7 @@ def not_excluded?(resource)
       end
 
       def page_ext
-        File.extname(app.index_file)
-      end
-
-      def register_extension_templates
-        # We call reload_path to register the templates directory with Middleman.
-        # The path given to app.files must be relative to the Middleman site's root.
-        templates_dir_relative_from_root = Pathname(TEMPLATES_DIR).relative_path_from(Pathname(app.root))
-        app.files.reload_path(templates_dir_relative_from_root)
```

Comment:
It seems to work fine without it. The Middleman API no longer has `app.files.reload_path` as far as I can tell.